netbox_cmdb/netbox_cmdb/views.py

```diff
@@ -68,98 +69,171 @@ def get(self, request, *args, **kwargs):
         Args:
             request: The current request
         """
-        obj = self.get_object(**kwargs)
+        object = self.get_object(**kwargs)
         form = ConfirmationForm(initial=request.GET)
-
-        # If this is an HTMX request, return only the rendered deletion form as modal content
-        if is_htmx(request):
-            # form_url = reverse("decommisioning_delete", kwargs={'pk': obj.pk})
-            form_url = f"/plugins/cmdb/decommisioning/{kwargs['pk']}/delete"
-            return render(
-                request,
-                "htmx/delete_form.html",
-                {
-                    "object": obj,
-                    "object_type": self.queryset.model._meta.verbose_name,
-                    "form": form,
-                    "form_url": form_url,
-                    **self.get_extra_context(request, obj),
-                },
-            )
+        devices = []
+
+        if isinstance(object, Device):
```

Comment:
Wouldn't it be better to have a View class for each object type (if that's possible) instead of doing if/elif ?

Reply:
you are totally right, I even already have the dedicated view...
on it

Reply:
done